lib/polarshift/request.rb

```diff
@@ -374,7 +379,8 @@ def wait_op(url: nil, id: nil, timeout: 360)
           when "queued", "running"
             next
           when "failed"
-            raise "PolarShift operation failed:\n#{res["error"]}"
+            logger.warn "PolarShift operation failed:\n#{res["error"]}"
```

Comment:
what about using an error indicator than warning?

Reply:
It's indeed an error status ( a `failed` status from Polarion) initially.
As we are retrying when met this, and it's very likely to pass within 5 retries based my testing locally. So we finally passed this steps and the CI move on. I think a warn would be more appropriate here.
Of course, an error here is also acceptable to me. Let me know which one do you prefer ?

Reply:
OK, thanks for the clarification